### ISSUE TITLE
chore(dependencies): Security issue

### ISSUE THREAD

**Issue by dalisoft** [stoplightio/spectral#1746]
**Chore summary**
https://github.com/advisories/GHSA-35jh-r3h4-6jhm

**Tasks**
- [ ] Upgrade `lodash` to version 4.17.21 or later

**Additional context**

```diff
❯ yarn why lodash
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "lodash"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "lodash@4.17.21"
info Has been hoisted to "lodash"
info Reasons this module exists
   - "workspace-aggregator-9d7c4e81-a142-402a-a0aa-d73d18c6433a" depends on it
   - Hoisted from "_project_#@commitlint#cli#lodash"
   - Hoisted from "_project_#@commitlint#cli#@commitlint#load#lodash"
   - Hoisted from "_project_#smartlint#stylelint#lodash"
   - Hoisted from "_project_#smartlint#dockerfilelint#lodash"
+   - Hoisted from "_project_#smartlint#@stoplight#spectral#@stoplight#json#lodash"
   - Hoisted from "_project_#commitlint-config-airlight#@commitlint#config-conventional#conventional-changelog-conventionalcommits#lodash"
   - Hoisted from "_project_#@commitlint#cli#@commitlint#read#git-raw-commits#lodash"
   - Hoisted from "_project_#@commitlint#cli#@commitlint#load#@commitlint#resolve-extends#lodash"
   - Hoisted from "_project_#lerna#@lerna#create#whatwg-url#lodash"
+   - Hoisted from "_project_#smartlint#@stoplight#spectral#@stoplight#json-ref-resolver#@stoplight#json#lodash"
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#conventional-commits#conventional-changelog-core#lodash"
   - Hoisted from "_project_#@commitlint#cli#@commitlint#lint#@commitlint#parse#conventional-commits-parser#lodash"
   - Hoisted from "_project_#@commitlint#cli#@commitlint#lint#@commitlint#rules#@commitlint#ensure#lodash"
   - Hoisted from "_project_#lerna#@lerna#clean#@lerna#prompt#inquirer#lodash"
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#conventional-commits#conventional-changelog-core#conventional-changelog-writer#lodash"
info Disk size without dependencies: "4.88MB"
info Disk size with unique dependencies: "4.88MB"
info Disk size with transitive dependencies: "4.88MB"
info Number of shared dependencies: 0
=> Found "@stoplight/spectral#lodash@4.17.20"
info This module exists because "_project_#smartlint#@stoplight#spectral" depends on it.
info Disk size without dependencies: "4.86MB"
info Disk size with unique dependencies: "4.86MB"
info Disk size with transitive dependencies: "4.86MB"
info Number of shared dependencies: 0
✨  Done in 0.62s.
```

**Comment by toby-murray-snow-software** [comment]
Just as an FYI, Dependabot has been complaining about this as "high severity" vulnerability:

![image](https://user-images.githubusercontent.com/65908556/125479362-b23f9eea-7a93-43f5-b41b-3ac830a20807.png)

**Comment by kilahm** [comment]
Looks like #1585 solves this?

**Comment by dalisoft** [comment]
@kilahm Yes, I don’t understand why it’s still shows vulnerable?

**Comment by toby-murray-snow-software** [comment]
my bad - looks like an `npm update` does resolve this locally. Dependabot is satisfied, I misread the warning and assumed it was referring to a different project that it actually does.

**Comment by titanve** [comment]
Hello, 

I'm still getting the warning from `dependabot` about the `lodash: 4.17.20`. Can you point me in the right direction to fix this?
![image](https://user-images.githubusercontent.com/4803351/125851206-ae8f3362-37a7-4b29-81b3-ca696f8cfe01.png)


Thank you!

**Comment by dalisoft** [comment]
@titanve As a temporary workaround you can try [this](https://github.com/dalisoft/airlight/commit/a73b00f0b01ada9564fd85813754e31c3c9cbdb4#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R37) if you using `yarn`

**Comment by titanve** [comment]
@dalisoft thank you!

**Comment by hiddewie** [comment]
Could #1585 be released in a 5.x patch version? Thanks in advance!

**Comment by P0lip** [comment]
Fixed in 5.9.2.
My apologies for such a long waiting time.
We were all busy with v6.
